src/views/ApartmentApp/components/StaffMenu/components/ApplicationTable/components/ApplicationRow/applicationRow.scss

```diff
@@ -0,0 +1,6 @@
+@import '../../../../../../../../vars';
+
+.apartment-sub-table {
+  padding-top: 0 !important;
+  padding-bottom: 0 !important;
```

Comment:
The `!important` css property is a major code smell. Anywhere it is used, you should be able to achieve the same effect by simply adding more specificity to the selector (e.g. by adding more classnames, prepending the html element the class is applied to, etc.).

Creating css rules with !important means that those rules can only be over-written by other rules marked !important with a higher specificity. In this case, it isn't likely to matter, but as a rule it's bad practice.

Reply:
I'll look into that. The Material-UI things tend to be very particular and stubborn unless you use inline styles like they do in the Material-UI documentation. 
They have a prop for `class` that can be used to easily override the default component styling, but that feature does not work with styles defined in css/scss, only for the inline objects with useStyle and such.

Reply:
At the time that I made that style change, I had no idea how else to get it to work, but I have since learned how to override the Material UI things when I needed the make the striped/alternative colors for the table rows

Reply:
The main reason that Material-UI styles tend to override ours is that they're loaded last, and thus take precedence over our bespoke css files that are loaded before the MUI ones. 

The two easiest ways to override them are to either: 1) include the html element in the selector (e.g. `div.apartment-sub-table`) or 2) include one of the MUI classes on the element you're selecting (e.g. `.MuiGrid-root.apartment-sub-table`). Both of those selectors will have higher specificity than the MUI ones (which in my experience only include one class), and so will take precedence.

Reply:
Ah, it seem the `span.apartment-sub-table` style would be the easier solution, since it is much easier to check which HTML type the thing is than getting the MUI root, since there are many MUI sub-types in the styles and I always end up just trying them and seeing which one makes it work the way that I wanted

Additionally, using the `div.custom-class` , `span.custom-class` form makes the style file much easier to read